app/controllers/PreferredReplicaElection.scala

```diff
@@ -85,4 +86,103 @@ class PreferredReplicaElection (val messagesApi: MessagesApi, val kafkaManagerCo
       )
     }
   }
+
+  def handleScheduledIntervalAPI(cluster: String): Action[AnyContent] = Action.async { implicit request =>
+    featureGate(KMScheduleLeaderElectionFeature) {
+      val interval = kafkaManager.pleCancellable.get(cluster).map(_._2).getOrElse(0)
+      Future(Ok(Json.obj("scheduledInterval" -> interval))
+        .withHeaders("X-Frame-Options" -> "SAMEORIGIN"))
+    }
+  }
+
+  def scheduleRunElection(c: String) = Action.async { implicit request =>
+    var status_string: String = ""
```

Comment:
Suggestion: Refrain to use var to avoid side effects, either use a pattern match or [functor](https://blog.knoldus.com/functors/) EITHER.